packages/patternfly-4/react-core/src/components/Breadcrumb/BreadcrumbHeading.d.ts

```diff
@@ -0,0 +1,13 @@
+import { SFC, HTMLProps, ReactNode, ReactType } from 'react';
+
+export interface BreadcrumbHeadingProps extends HTMLProps<HTMLDivElement> {
+    children?: ReactNode;
+    className?: string;
+    to?: string;
+    target?: string;
+    component?: ReactType<BreadcrumbItemProps>;
```

Comment:
I've never heard of `ReactType` maybe you're talking about `ReactElement`?

Reply:
We are using `ReacType` in all components where a component prop is used.